### ISSUE TITLE
When installation fails there is no reinstall option to repair it

### ISSUE THREAD

**Issue by Ray-Hoo** [fr3nd/msxhub#18]
I tried to install the musicpac several times, and it ends in an HTTP error 400 several times. Every time I have to remove the installation and then start all over again. With a reinstall option, hub should check if the files installed are the same as on the server and then skip them.

**Comment by ducasp** [comment]
@Ray-Hoo , if I'm not mistaken you can use UPGRADE, it always uninstall whatever is installed with that package name and re-install it.

**Comment by Ray-Hoo** [comment]
Yes, I know you can use upgrade. However, the process starts from the beginning again and when you hit another HTTP error 400 you can start all over again. And that is where the reinstall option comes in, when the files are there (compared on size and date/timestamp) then it can be skipped, which makes it faster.
